eland/cli/eland_import_hub_model.py

```diff
@@ -237,13 +237,13 @@ def check_cluster_version(es_client, logger):
         )
         exit(1)
 
-    # PyTorch was upgraded to version 2.3.1 in 8.15.2
+    # PyTorch was upgraded to version 2.5.1 in 8.18.0
     # and is incompatible with earlier versions
-    if sem_ver < (8, 15, 2):
+    if sem_ver < (8, 18, 0):
         import torch
 
         logger.error(
-            f"Eland uses PyTorch version {torch.__version__} which is incompatible with Elasticsearch versions prior to 8.15.2. Please upgrade Elasticsearch to at least version 8.15.2"
+            f"Eland uses PyTorch version {torch.__version__} which is incompatible with Elasticsearch versions prior to 8.18.0. Please upgrade Elasticsearch to at least version 8.18.0"
```

Comment:
Why do this is PyTorch is backwards-compatible, then?

Reply:
I didn't express that very clearly let me try again.

For a model traced in Eland using an earlier version of PyTorch, that model can be evaluated in a later version of PyTorch in Elasticsearch. It is BWC (from the Elasticsearch perspective) in that a model created in an older version of PyTorch can run in newer versions. The ELSER model was created in an earlier version of PyTorch and continues to run in the new versions after upgrade.

A model traced in a new version of PyTorch may use features that are not available in an older version of the runtime and cannot run in an Elasticsearch cluster that uses an older version.

Reply:
Right, this is obvious now, thank you!